### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by keraliss**

### CODE REVIEW

**Comment by keraliss** [comment]
Screenshot from 2024-09-28 16-15-29
Screenshot from 2024-09-28 16-15-34
Screenshot from 2024-09-28 16-15-38
Screenshot from 2024-09-28 16-15-41
Screenshot from 2024-09-28 16-15-44
Screenshot from 2024-09-28 16-15-52
Screenshot from 2024-09-28 16-15-58
Screenshot from 2024-09-28 16-16-02
Screenshot from 2024-09-28 16-16-05
Screenshot from 2024-09-28 16-16-08
Screenshot from 2024-09-28 16-16-11
Screenshot from 2024-09-28 16-16-14
Screenshot from 2024-09-28 16-16-45
Screenshot from 2024-09-28 16-16-52
Screenshot from 2024-09-28 16-16-55
Screenshot from 2024-09-28 16-16-57
Screenshot from 2024-09-28 16-17-01
Screenshot from 2024-09-28 16-17-08
Screenshot from 2024-09-28 16-17-17
Screenshot from 2024-09-28 16-17-23
Screenshot from 2024-09-28 16-17-26
Screenshot from 2024-09-28 16-17-29

**Comment by keraliss** [comment]
pexels-ingewallu-126407

**Comment by keraliss** [comment]
Screenshot from 2024-09-28 16-26-07

**Comment by keraliss** [comment]
Screenshot from 2024-09-30 12-47-09

File: src/components/cohortPages/Description.astro [hunk 1]
```diff
@@ -15,7 +15,9 @@ const { takeaways } = Astro.props;
       {
         takeaways.map((takeaway) => (
           <li class="my-4 lg:text-xl">
-            <span class="font-header">{takeaway.name}</span>{" "}
+            <span class="font-header font-bold">
+              {takeaway.name}
+            </span>{" "}
```
Comment (emjshrx) [inline src/components/cohortPages/Description.astro:20]: is the `{" "}` needed

File: src/components/common/list.js [hunk 2]
```diff
@@ -209,3 +209,60 @@ export const reviewClubTa = [
...
+      name: "Bala",
+      cohort: "on PB Cohort",
+      img: "/testimonial/bala.png",
+    },
+    // {
+    //   text: "Personally, I was able to familiarize myself more deeply with concepts I only knew at a surface level. I was also able to get myself up-to-date with many recent developments. Additionally, a special shoutout goes to the other weekly clubs conducted by Bitshala that I attended alongside the cohort — Bitcoin Optech Club, Reading Club, and the PR Review Club. They covered a ton of additional information, and attending them is a must for anyone who wants to stay up-to-date with everything happening in Bitcoin. I can assure you that anyone who is able to stick through the entire Mastering Bitcoin cohort will gain a better understanding of Bitcoin than 99% of people and will be in a good position to grab opportunities in the Bitcoin space or start something of their own.",
+    //   name: "Claddy",
+    //   cohort: "on Mastering Bitcoin Cohort",
+    //   img: "/testimonial/claddy.png",
```
Comment (emjshrx) [inline src/components/common/list.js:231]: remove comment

File: src/components/home/About.astro [hunk 3]
```diff
@@ -1,43 +1,140 @@
...
+            <img
+              src={li.link}
+              class="h-8 object-contain lg:h-16"
+              alt="Bitshala Logo"
+            />
+          </a>
+        ))
+      }
+    </div>
+  </div> -->
```
Comment (emjshrx) [inline src/components/home/About.astro:139]: remove comment

File: src/components/home/Testimonial.astro [hunk 4]
```diff
@@ -1,89 +1,55 @@
...
+import TestimonialMobile from "./TestimonialMobile";
+
+const imageList = [
+  ["/testimonial/1.webp", "/testimonial/2.webp"],
+  [
+    "/testimonial/3.webp",
+    "/testimonial/4.webp",
+    "/testimonial/5.webp",
+  ],
+  ["/testimonial/6.webp", "/testimonial/7.webp"],
```
Comment (emjshrx) [inline src/components/home/Testimonial.astro:12]: Pls make sure you remove all old images

File: src/components/readingClub/PreviousSessions.jsx [hunk 5]
```diff
@@ -46,7 +48,7 @@ const PreviousSessions = ({
                 ></iframe>
 
                 <div class="my-2 flex flex-col lg:mx-5">
-                  <h1 class="font-header text-2xl text-orange lg:mb-5 lg:text-3xl">
+                  <h1 class="font-header font-bold text-2xl text-orange lg:mb-5 lg:text-3xl">
                     {video.title.slice(0, 55)}...
```
Comment (emjshrx) [inline src/components/readingClub/PreviousSessions.jsx:52]: remove comments

File: src/pages/index.astro [hunk 6]
```diff
@@ -21,45 +24,52 @@ const isVisible = true;
...
-  <LinkCohort />
-  <People />
-  <Newsletter client:load />
-  <Calender />
+  <Testimonial />
+  <JourneyGuide />
+  <LinkCohort class:load />
+  <!-- <People /> -->
+  <!-- <Newsletter client:load /> -->
+  <!-- <Calender /> -->
```
Comment (emjshrx) [inline src/pages/index.astro:74]: Remove comments

File: src/pages/review.astro [hunk 7]
```diff
@@ -58,7 +58,7 @@ const isVisible = true;
 >
   <div slot="headerChild" class="mt-8 lg:mt-16">
     <h1
-      class="text-center font-header text-4xl text-white md:text-6xl lg:text-8xl"
+      class="text-center font-header text-4xl font-bold text-white md:text-6xl lg:text-8xl"
```
Comment (emjshrx) [inline src/pages/review.astro:61]: If font-bold is added to all headers you could have just added a common style to all headers in the global CSS file

File: src/components/cohortPages/Description.astro (hunk 1 above)
Comment (keraliss) [reply src/components/cohortPages/Description.astro:20]: We currently are in the middle of design change, where one cohort page is redesigned, but others are not. once all the pages are redesigned and have a similar layout, we can remove duplicate components and use one for all

File: src/components/home/Testimonial.astro (hunk 4 above)
Comment (keraliss) [reply src/components/home/Testimonial.astro:12]: These are new images for the testimonial section

File: src/components/home/TestimonialMobile.jsx [hunk 8]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/css/navigation";
+import "swiper/css/pagination";
+import "swiper/swiper-bundle.css";
```
[the remark this answers is not in the document]
Comment (keraliss) [reply src/components/home/TestimonialMobile.jsx:5]: we already have swiper in our package as well. we were using it in previous components as well. and this provides a very fluent carousel experience. so i would prefer swiper over mui carousel components if possible

File: src/pages/review.astro (hunk 7 above)
Comment (keraliss) [reply src/pages/review.astro:61]: yes,can do that. but the issue is not just with header. previously we just had one font size for header, now we need to switch between bold and medium. that's why i had to add so many font-bolds at once

**Review by emjshrx — APPROVED** [review]
looks good if the comments and logs are removed

**Pull request merged**